node/src/Commands.ts

```diff
@@ -2440,9 +2440,6 @@ export type StreamClaimOptions = {
      * otherwise the IDs of non-existing messages are ignored.
      */
     isForce?: boolean;
-
-    /** The last ID of the entry which should be claimed. */
```

Comment:
I wasn't able to get LASTID to do this for me through testing.  I don't know if it actually works that way.  
Removing this is the correct way forward until Valkey properly documents and tests this.